frontend-student/src/components/signup/Signup.jsx

```
@@ -0,0 +1,140 @@
+      handleSignUp(values);
+    },
+  });
+
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen">
+      <div className="w-[400px] p-6 bg-white shadow-md rounded-lg">
+        <h1 className="text-2xl font-bold text-center">Sign Up</h1>
+        <p className="text-gray-600 text-center">Create a new account</p>
+        <form onSubmit={signupFormik.handleSubmit} className="mt-4 space-y-3">
+          <div>
+            <label className="block text-gray-700 font-medium">Name</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)